src/components/__tests__/drop_down_menu_base.test.tsx

```diff
@@ -66,7 +67,8 @@ describe('The DropDownMenuBase component', () => {
     });
 
     it('should call toggle menu handler when escape is pressed', () => {
-        wrapper.instance()._onOutsideFocusHandler({ key: 'Escape' });
+        // TODO this needs a better way to test, along with the click event, as right now it is not meaningful.
+        wrapper.instance()._keyPressHandler({ key: 'Escape' });
```

Comment:
Have seen a few tests that are actually mocking the user "click" buttons to perform actions. One example:
https://github.com/Ultimaker/stardust-mycloud/blob/staging/src/views/__tests__/ClustersView.test.tsx#L229

Would it be possible to implement something similar, so that we can compare the components that are in the DOM, before and after the updates?

Reply:
Perhaps it is meaningful to illustrate why `as right now it is not meaningful`?

Reply:
This is different, because you have to simulate a click somewhere on the document, outside this component. It is attaching a listener to the document, through which you'd have to trigger a method.
Here it is not meaningful, because a private method is just called on the component. If I add the proper type to the wrapper, this is not even possible.

Reply:
Added some more details to TODO.